api.ts

```diff
@@ -1521,6 +1521,11 @@ export type PropertyOptionsResults = PropertyOptionsMetadataResult<any | Propert
 interface PropertyOptionsNormalizedResults {
   cacheTtlSecs?: number;
   results: Array<{display: string | undefined; value: any}>;
+
+  // Undocumented feature for situations when you might be using an awkward API that
+  // only lets you fetch data using an overly-specific property when a more general
+  // one would have sufficed.
+  propertiesNotUsed?: string[];
```

Comment:
A quick brainstorm of other name options:
- unusedProperties
- cacheIgnoreProperties
- cacheKeyExcludeProperties

And one other possibility - instead of a negative framing, we could in theory make this `cacheKeyProperties` and have it override our property spying completely. Perhaps easier to explain/reason about? Perhaps harder for the pack maker to keep accurate as the pack evolves?

Very curious for @ekoleda-codaio 's thoughts.

Reply:
I like the conceptual simplicity of a positive framing like `cacheKeyProperties` as a way to opt out of implicit caching entirely, although I'm also nervous that if the only way to ignore a key is opting out of implicit caching entirely it significantly raises the technical skills needed to just ignore one or two properties

My inclination for now is to go with something like `unusedProperties` for simplicity while relying on the fact that this is will be an undocumented feature that we can always change later. Open to a different approach, though.

Reply:
Sounds good to me.

Reply:
Cool, renamed the undocumented setting to `unusedProperties`